app/assets/javascripts/ckeditor/plugins/cortex_media_insert/plugin.js

```diff
@@ -0,0 +1,34 @@
+(function (global) {
```

Comment:
`media_select_defer` only becomes globally accessible because we set this here, otherwise `global.X` means nothing